schema/TRIB_REC.json

```diff
@@ -286,7 +286,7 @@
       "description": "",
       "type": "number",
       "minimum": 0,
-      "maximum": 999999999999,
+      "maximum": 999999999.99,
```

Comment:
I think the max still needs to be 999999999999.  It's kind of funny but we're only limited by the .FEC specs which say the number has to be represented in 12 or less characters as a string.  So .FEC can have a value of 1000000000 which is higher than 999999999.99 but takes less characters.

Reply:
Per our online Slack conversation, for now we're going to go with sub-1-billion values but follow up with requirements to see if there is a higher dollar value limit since fecfile online is not limited by the string formatting of the print form but rather by the WebCheck limits and FEC database limits.